app/pulp/app/models/progress.py

```diff
@@ -82,9 +82,8 @@ def __enter__(self):
         """
         Saves the progress report state as RUNNING
         """
-        if self.state != self.RUNNING:
-            self.state = self.RUNNING
-            self.state.save()
+        self.state = self.RUNNING
+        self.state.save()
```

Comment:
Should this just be `self.save()`? `self.state` isn't a relation, so there's nothing to save there.